demo/index.html

```diff
@@ -17,6 +17,8 @@
 
         <!-- css files -->
         <link rel="stylesheet" id='css-theme' href="../build/css/chamel-base.css" />
+        <link rel="stylesheet" id='css-codemirror' href="../node_modules/codemirror/lib/codemirror.css" />
```

Comment:
Can we include both of these into our sass so the user does not have to add them as dependencies? I would put them in scss/components/editor/vendor/_codemirror.scss and scss/components/editor/vendor/_colorpickr.scss